### DISCUSSION TITLE
toc-drawer no toc on index.html in versions after 2022.4.7

### DISCUSSION THREAD

**Discussion by KRSatpute** [pradyunsg/furo#457] [Potential Issue] [answered]
When index.html is generated, if there is some content in the file right side toc-drawer creates a neat toc-tree-container.

This is missing in versions after 2022.4.7
In versions after 2022.4.7 we get toc-drawer no-toc. 

The toc-drawer toc-tree-container works properly in latest version for when we navigate to other pages in generated documentation.

![image](https://user-images.githubusercontent.com/33159809/174468732-9d5b4ef8-0f92-4f69-880b-69b6d882793e.png)

![image](https://user-images.githubusercontent.com/33159809/174468754-79fbb56a-3e81-48c4-a142-b3a16888de7b.png)

**Comment by pradyunsg** [comment]
Could you provide a reproducer for this issue? A documentation set, instructions on how to build it and Sphinx + Furo versions you’re using would be sufficient. :)

**Reply by pradyunsg** [reply, marked as answer]
Closing this given the lack of a response.
